docs/src/doc/user-guide/advanced/kotlin-singleton.md

```diff
@@ -1,7 +1,21 @@
-Sometimes you need to store some Godot objects or references in a Kotlin singleton. This can cause some leaks when the program ends because you have to manually free objects and have the references collected. The `GodotStatic` interface lets you implement the function `collect` so you can write the appropriate cleanup code. Then you just need to call `registerAsSingleton` so it will be hooked to the Garbage Collector. 
+Sometimes, you need to store some Godot objects or references in a Kotlin singleton. This can cause some leaks when the program ends because you have to manually free objects and have the references collected. The `GodotStatic` abstract class lets you implement the function `collect` so you can write the appropriate cleanup code.
```

Comment:
```suggestion
Sometimes you need to store some Godot objects or references in a Kotlin singleton, this can cause some leaks when the program ends because you have to manually free objects and have the references collected. The `GodotStatic` abstract class lets you implement the function `collect` so you can write the appropriate cleanup code.
```